src/Bigcommerce/Api/Client.php

```diff
@@ -2040,4 +2040,15 @@ public static function deleteShippingMethod($zoneId, $methodId)
     {
         return self::deleteResource('/shipping/zones/'. $zoneId . '/methods/'. $methodId);
     }
+    
+    /**
+    Get collection of product skus by Product
```

Comment:
Can you format this like:
```
    /**
     * Get collection of product skus by Product
     * @param array $filter
     * @return mixed
     */
```

That way it'll get picked up as a docblock in IDEs like PHPStorm.